client/src/components/searchSidebar/style.css

```diff
@@ -0,0 +1,67 @@
+.container {
+  background: #363936;
+  left: 80px;
+  height: 100%;
+  width: 21vw;
+  position: fixed;
+}
+
+.search-sidebar {
+  display: flex;
+  flex-direction: column;
+  width: 21vw;
+  position: fixed;
+  top: 29%;
+}
+
+
+.searchform {
+  height: 4vh;
```

Comment:
use either vh everywhere or % everywhere. Using both might lead to unexpected css bugs.